### DISCUSSION TITLE
IPv6 address for rspamd.com

### DISCUSSION THREAD

**Discussion by dwydler** [rspamd/rspamd#5124] [General]
Hello everyone,
This afternoon I wanted to install the Rspamd package via apt on a server that only has an IPv6 address.
Unfortunately I get the following error message:
```
Err:2 http://rspamd.com/apt-stable noble Release
Cannot initiate the connection to rspamd.com:80 (135.181.136.158). - connect (101: Network is unreachable)
```
Are there any understandable reasons why the web server only has an IPv4 address ? Especially since Hetzner already supports DualStack on root servers.

Best regards,
Daniel

**Comment by vstakhov** [comment]
Are there any "understandable reasons" why can't you use NAT64 for that? If no, then please close this discussion.

**Comment by dwydler** [comment]
A crutch for a crutch... Why should I implement NAT64 when the Hoster officially offers Dual Stack for dedicated servers?

**Comment by vstakhov** [comment]
I'd say that dual stack is a crutch. And it has caused a lot of troubles for many other users in the past. Hence, it won't be enabled. I'm fine to add ipv6 only alias for the site and other resources - it won't hurt anyone.
